pkg/handler/processor/process/process.go

```diff
@@ -40,12 +45,57 @@ func RegisterDocumentProcessor(p processor.DocumentProcessor, d processor.Docume
 	return nil
 }
 
```

Comment:
can probably be in its own package?

Reply:
breakout the `Subscribe` to be its own package?